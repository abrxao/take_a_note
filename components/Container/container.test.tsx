import { act, fireEvent, render, screen } from "@testing-library/react";

import Container from ".";

describe("Note card tests", () => {
  test("render skeleton", async () => {
    render(<Container>Container</Container>);
    const noteCardSkeleton = screen.getByText("Container");

    expect(noteCardSkeleton).toMatchSnapshot();
  });
});
