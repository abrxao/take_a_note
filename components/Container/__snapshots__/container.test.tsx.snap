// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Note card tests render skeleton 1`] = `
<div
  class="px-3 xs:px-4 py-3 lg:py-5 w-full lg:px-12 xl:px-16 2xl:px-24"
>
  Container
</div>
`;
